msibi/optimize.py

```diff
@@ -1,3 +1,34 @@
+##############################################################################
+# MSIBI: A package for optimizing coarse-grained force fields using multistate
+#   iterative Boltzmann inversion.
+# Copyright (c) 2017 MIT University and the Authors
+#
+# Authors: Christoph Klein
```

Comment:
Sorry about that. I changed the Authors and Collaborators section according to the format in the setup.py file. Will update to reflect requests.